nltk/translate/meteor_score.py

```diff
@@ -38,10 +49,10 @@ def exact_match(hypothesis, reference):
     and returns a word mapping based on the enumerated
     word id between hypothesis and reference
 
-    :param hypothesis: hypothesis string
-    :type hypothesis: str
-    :param reference: reference string
-    :type reference: str
+    :param hypothesis: pre-tokenized hypothesis
+    :type hypothesis: array-like(str)
+    :param reference: pre-tokenized reference
+    :type hypothesis: array-like(str)
```

Comment:
```suggestion
    :type reference: array-like(str)
```
I believe this is a small copy-paste error. No biggie.